packages/legend-graph/src/graph-manager/action/service/ServiceExecutionServer.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export enum ServiceExecutionServer {
+  INT = 'INT',
```

Comment:
you don't need this enum. You can get this info from the config `TEMPORARY__serviceRegistrationConfig`

Reply:
yes agree we shouldnt need to fetch it.

Reply:
removed! use a const instead